<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>823</width>
    <height>493</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_4">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_4">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="styleSheet">
          <string notr="true">font: 16pt &quot;Adobe 黑体 Std R&quot;;</string>
         </property>
         <property name="text">
          <string>人脸识别</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="comboBox_1">
         <property name="editable">
          <bool>true</bool>
         </property>
         <property name="placeholderText">
          <string>input</string>
         </property>
         <item>
          <property name="text">
           <string>arcface</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>facenet</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="comboBox_2">
         <property name="editable">
          <bool>true</bool>
         </property>
         <property name="placeholderText">
          <string>input</string>
         </property>
         <item>
          <property name="text">
           <string>摄像头</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>图片</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>视频</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="select">
         <property name="text">
          <string>select</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="start">
         <property name="text">
          <string>start</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="stop">
         <property name="text">
          <string>stop</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="show2">
         <property name="text">
          <string>show</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="upload">
         <property name="text">
          <string>upload</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="styleSheet">
          <string notr="true">font: 18pt &quot;Adobe 黑体 Std R&quot;;</string>
         </property>
         <property name="title">
          <string>input</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout">
            <item>
             <widget class="QLabel" name="input">
              <property name="enabled">
               <bool>true</bool>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="margin">
               <number>-1</number>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="Line" name="line">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_2">
         <property name="styleSheet">
          <string notr="true">font: 18pt &quot;Adobe 黑体 Std R&quot;;</string>
         </property>
         <property name="title">
          <string>output</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <item>
             <widget class="QLabel" name="output">
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
